app/controller.js

```diff
@@ -59,8 +58,22 @@ exports.getComments = (req,res,next) =>{
       
         return res.status(200).send({comments:rows})
 
-    }).catch(({code})=>{if(code==="22P02"){
-        return res.status(400).send({msg:"Invalid query params!!"})
-        }else{code}
+    }).catch(({code})=>{
+        if(code){
+        next(code)}
+    })
+}
+
+exports.postComment = (req,res,next) =>{
+    const comment = req.body
+    const articleId = req.params
+    
+    updateComments(comment,articleId).then(({rows})=>{
+        return res.status(201).send({comment:rows})
+    }).catch(({code})=>{
+        if(code === undefined){return res.status(400).send({msg:"Invalid username"})}
```

Comment:
this should also be error not code, and you don't need to check if it's undefined here, just pass the error to your error handling middleware to deal with please!